packages/blade/src/components/Input/DropdownInputTriggers/AutoComplete.stories.tsx

```diff
@@ -25,7 +25,7 @@ const propsCategory = {
 };
 
 export default {
-  title: 'Components/Dropdown/With AutoComplete',
+  title: 'Components/Dropdown/With AutoComplete/Stories',
```

Comment:
Its hard to discover because its called "Stories". Can we rename the nested folder itself to "Props Playground"? Would be easier to discover

```suggestion
  title: 'Components/Dropdown/With AutoComplete/Props Playground',
```

Reply:
best!

Reply:
done